sdk/storage/azure-storage-blob/azure/storage/blob/_shared/encryption.py

```diff
@@ -101,11 +131,20 @@ class _EncryptionData:
     Represents the encryption data that is stored on the service.
     '''
 
-    def __init__(self, content_encryption_IV, encryption_agent, wrapped_content_key,
-                 key_wrapping_metadata):
+    def __init__(
+            self,
+            content_encryption_IV,
+            encrypted_region_info,
+            encryption_agent,
+            wrapped_content_key,
+            key_wrapping_metadata):
         '''
-        :param bytes content_encryption_IV:
+        :param Optional[bytes] content_encryption_IV:
             The content encryption initialization vector.
+            Required for AES-CBC.
```

Comment:
Nit: After looking futher in the review, I see that the correspondence for selecting between AES-CBC and AES-GCM is whether `version` is V1 or V2. May be helpful to sort of make this obvious to cx when referring to the algorithms, something like this:
AES-CBC (V1)
AES-GCM (V2)
So it is painfully obvious whenever you decide to refer to V1 or V2 what that exactly means!

Reply:
Sure, I can do that. This is an internal only method that a customer would never see/call but I can add it to help us remember :)